packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -123,7 +176,27 @@ export class TestRunnerService {
 				continue;
 			}
```

Comment:
In which case might this not be defined? Maybe a comment could clarify this

Reply:
Added a comment